Library/Formula/rem.rb

```diff
@@ -0,0 +1,35 @@
+# Documentation: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md
+#                /usr/local/Library/Contributions/example-formula.rb
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
+
+class Rem < Formula
+  desc "Command-line tool to access OSX Reminders.app database"
+  homepage "https://github.com/kykim/rem"
+  url "https://github.com/kykim/rem/archive/20150618.tar.gz"
+  version "20150618"
+  sha256 "e57173a26d2071692d72f3374e36444ad0b294c1284e3b28706ff3dbe38ce8af"
+
+  # depends_on "cmake" => :build
+
+  def install
+    # ENV.deparallelize  # if your formula fails when building in parallel
+
+    # Remove unrecognized options if warned by configure
+    system "xcodebuild"
+    system "mkdir", "#{prefix}/bin"
+    system "cp", "build/Release/rem", "#{prefix}/bin"
+  end
+
+  test do
+    # `test do` will create, run in and delete a temporary directory.
+    #
+    # This test will fail and we won't accept that! It's enough to just replace
+    # "false" with the main program this formula installs, but it'd be nice if you
+    # were more thorough. Run the test with `brew test rem`. Options passed
+    # to `brew install` such as `--HEAD` also need to be provided to `brew test`.
+    #
+    # The installed folder is not in the path, so use the entire path to any
+    # executables being tested: `system "#{bin}/program", "do", "something"`.
+    system "#{bin}/rem", "version"
```

Comment:
A more substantial test would be great; something that checks general functionality.